tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -135,4 +139,17 @@ func testFilesCopied(t *testing.T) {
 		assert.Emptyf(t, stdout.String(), "Missing file: %s", fullPath)
 	}
 
+	// Check the SHA of kube.tar.gz downloaded
+	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
+	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
```

Comment:
Do you assume that `certutil` exist? If not, where do you capture the error?

Reply:
As I mentioned above, looks like certutil should exists for all Windows Server 2019 images. For now I can capture both stdout and stderr in client.Run command in case the execution fails